sdcm/reporting/tooling_reporter.py

```diff
@@ -98,6 +102,31 @@ def _collect_version_info(self) -> None:
                 driver_version=driver_version, command_prefix=None, runner=None, argus_client=self.argus_client).report()
 
 
+class CqlStressCassandraStressVersionReporter(ToolReporterBase):
+    # pylint: disable=too-few-public-methods
+    TOOL_NAME = "cql-stress-cassandra-stress"
+
+    def _collect_version_info(self) -> None:
+        output = self.runner.run(f"{self.command_prefix} {self.TOOL_NAME} version_json")
+        LOGGER.info("%s: Collected cql-stress-cassandra-stress output:\n%s", self, output.stdout)
```

Comment:
Maybe change it to the `debug` level?
We may have dozens of stress commands...

Reply:
I didn't think about it - just copied over from other implementations.
Sounds it should be debug - as anyway we use Argus for getting this info.

Reply:
those should be call once per command, so yes debug should be good enough

Reply:
fixed